server/aesdsocket-start-stop.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/sh
```

Comment:
@srki3050 don't check your server/aesdsocket file into your repository - this can cause problems.  You need to build/install the one from buildroot.